package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/main.js",
   "scripts": {
     "build": "tsc",
-    "test": "jest"
+    "test": "npm run build && jest"
```

Comment:
Added the build to the tests to prevent testing in an older built file since the tests import `dist/main`